indico/core/celery/templates/celery/celery_tasks.html

```diff
@@ -0,0 +1,55 @@
+{% from 'message_box.html' import message_box %}
+{% extends 'layout/base.html' %}
+
+{% block title %}
+    {% trans %}Tasks{% endtrans %}
+{% endblock %}
+
+{% block content %}
+    <div class="tasks">
+        {% call message_box('info', fixed_width=True) %}
+            {% if flower_url %}
+            {% trans %}Monitor Indico related tasks with flower <a class="flower-url" href="{{ flower_url }}" target="_blank">here</a>.{% endtrans %}
+            {% else %}
+                {% trans %}
+                    Indico uses Celery to manage tasks which can be monitored using Flower.<br>
+                    Configure the Flower URL in <span class="mono">indico.conf</span> to have a link displayed here.
+                {% endtrans %}
+            {% endif %}
+        {% endcall %}
+
+        <h2>Period tasks</h2>
+        <table class="i-table-widget">
+            <thead>
+                <tr>
+                    <th>{% trans %}Name{% endtrans %}</th>
+                    <th>{% trans %}Schedule{% endtrans %}</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for task in tasks %}
+                    <tr>
+                        <td>{{ task.name }}</td>
+                        <td>{{ render_task_schedule(task) }}</td>
+                    </tr>
+                {% endfor %}
+            </tbody>
+        </table>
+    </div>
+{% endblock %}
+
+{% macro render_task_schedule(task) %}
+    {% if not task.disabled %}
+        {% set schedule = task.custom_schedule or task.schedule %}
+        {% if schedule is instanceof timedelta %}
+            {% trans interval=schedule|format_human_timedelta %}every {{ interval }} {% endtrans %}
+        {% else %}
+            <span class="mono">{{ schedule }}</span>
+        {% endif %}
+            {% if task.custom_schedule %}
+                <span class="i-label warning">{% trans %}overridden{% endtrans %}</span>
```

Comment:
"custom schedule" might sound better. it's not clear what "overridden" means here unless you already know it